src/test/java/com/axis/jenkins/plugins/eiffel/eiffelbroadcaster/Mocks.java

```diff
@@ -0,0 +1,56 @@
+/*
+ The MIT License
+
+ Copyright 2015 Sony Mobile Communications Inc. All rights reserved.
+ Copyright 2021 Axis Communications AB.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in
+ all copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ THE SOFTWARE.
+ */
+
+package com.axis.jenkins.plugins.eiffel.eiffelbroadcaster;
+
+import com.rabbitmq.client.AMQP;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import mockit.Mock;
+import mockit.MockUp;
+
+/**
+ * Test mocks.
+ *
+ * @author Örjan Percy &lt;orjan.percy@sonymobile.com&gt;
+ * @author Magnus Bäck &lt;magnus.back@axis.com&gt;
+ */
+final class Mocks {
+    /** Stores received messages. */
+    public static final List<String> MESSAGES = new CopyOnWriteArrayList<>();
```

Comment:
No big deal, but only immutables should be uppercase, even if `final const`. If I had not already read this code in mq-notifier, I would have been surprised to see it changed in `EmittedEventsTest.` For similar reasons `final static Logger logger` is preferred over `final const Logger LOGGER`.

Reply:
Done.